src/wasm/include/web-ifc-geometry.h

```diff
@@ -1188,6 +1190,9 @@ namespace webifc
 			uint32_t capSize = profile.curve.points.size();
 			for (int i = 1; i < capSize; i++)
 			{
+				if (std::count(holesIndices.begin(), holesIndices.end(), i))
+					continue;
```

Comment:
Please add explicit brackets here, rest is good!

Small note: this is now doing a quadratic search over the holes, my assumption would be that there's never too many holes in a polygon, but could you add a comment that this would become slow for many holes? Thx